Automattic/jetpack
# Ensure callables are callables

Jetpack Sync maintains a list of default callables however this list can be filtered. We need to ensure that callables are in fact callables before being called with call_user_func.

#### Changes proposed in this Pull Request:
* add is_callable prior to call_user_func to suppress warnings when the callable isn't callable.

#### Does this pull request change what data or activity we track or use?
* NO

#### Testing instructions:

* Add a filter to `jetpack_sync_callable_whitelist` that adds a new entry for a function that does not exist.
* Unlock callables
* $cm = Automattic\Jetpack\Sync\Modules::get_module( 'functions' );
* $cm->unlock_sync_callable();
* Access the wp-admin, edit a post
* Verify no warning in logs 

#### Proposed changelog entry for your changes:
* Jetpack Sync - resolve warnings if callables are not callables.
